### PR TITLE
readme

### PR DESCRIPTION
**Pull request by kcbreuil**
@ErnieAtLYD moved this repo to wyncode, please let me know your thoughts and leo will make some tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,15 @@ Read the instructions thoroughly before starting your project.
...
-In your database design homework, we had you design a sailorScoutSchema!
+In your database design homework, we had you design a `sailorScoutSchema`!
 
-Now it's time to put it to good use :D
+Now we are going to put it to good use, build some routes and make some Sailor Scouts :D
 
-We have set up your server.js and some preliminary code to make a sailor model and create a sailor route.
+We have set up your server.js and some preliminary code to make a sailor model and create some sailor routes.
+
+We would like you to create 3 different sailor scouts of your choice with their specific properties found here:
```
Comment (ErnieAtLYD) [inline README.md:26]: when you say "We would like you to create 3 different sailor scouts..." do you mean we'd like to create three sailor scout API endpoints? Three different models? Something else?

Comment (kcbreuil) [reply README.md:26]: @ErnieAtLYD ah good question, we'd like them to create 3 different sailor documents--iterations of the sailor moon model via the post route. Check all three have been created by using the get route

Comment (ErnieAtLYD) [reply README.md:26]: @kcbreuil gotcha. so perhaps:

> We would like you to ~create 3 different sailor scouts of your choice~ write three different sailor scouts to your database with their specific properties found here: (wiki page)

Although that leads to another question:

> with their specific properties found here: (wiki page)
Didn't we already create a model for the sailor scouts though? So are they going to the wiki page and creating more properties there?

Comment (kcbreuil) [reply README.md:26]: We created the models, but the properties would be different, depending on the sailor scout they are creating. @ErnieAtLYD

Comment (ErnieAtLYD) [reply README.md:26]: @kcbreuil  ahhhhhhh. okay so their asks are to:
 * Create 3 new MongoDB collections

And to do that they have to:
 * Add properties to the model file they were given

Are they tweaking their GET/POST route as well? (sorry, I just wanna be 100% sure)

Comment (kcbreuil) [reply README.md:26]: @ErnieAtLYD Yes! no it's very helpful if I am not being clear on what we want them to do. Yes, we want them to create unique collections that follow the schema we already built for them--to do this they would have to complete the routes so they are working. Once they finish the routes, they would create the three documents in the sailor collection

**Pull request merged**